docs/edgeql/statements/update.rst

```diff
@@ -43,13 +43,25 @@ selected by *update-selector-expr* and, optionally, filtered by
     information.
 
 :eql:synopsis:`SET <shape>`
-    A :ref:`shape <ref_eql_expr_shapes_update>` expression with
-    the new values for the links of the updated object.
+    A :ref:`shape <ref_eql_expr_shapes_update>` expression with the
+    new values for the links of the updated object. There are three
+    possible assignment operations permitted within the ``SET`` shape:
 
     .. eql:synopsis::
 
         SET { <link> := <update-expr> [, ...] }
 
+        SET { <link> += <update-expr> [, ...] }
```

Comment:
The use of link here is somewhat misleading, some may think only links are allowed in the `UPDATE` shape.  What about properties?

Reply:
I suppose I could call it "name" instead or "field". The latter being a familiar generic reference to what we internally dub "pointers". At the end of the day it's just a placeholder, so "field" is probably what most people would intuitively understand best.